source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,560 @@ Filters Builders
+   :stub-columns: 1
+
+   * - Evaluation Method
+     - Matches
+      
+   * - ``mod()``
+     - documents where a modulo operation on the value of a field contain a specified result.
+
+   * - ``regex()``
+     - documents where values contain a specified regular expression.
+      
+   * - ``text()``
+     - documents which contain a specified full-text search expression.
+     
+   * - ``where()``
+     - documents which contain a specified JavaScript expression.
+
+The following example matches documents that have a ``color`` field
+containing the letter "p" in the ``paint_purchases`` collection:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin regexComparison
+   :end-before: end regexComparison
+
+The BSON renders as:
+
+.. code-block:: json
+   :copyable: false
+   
+   { "$regex" : "p" }
+
+The output of the above code snippet should look something like this:
+   
+.. code-block:: json
+   :copyable: false
+      
+   {"_id": 2, "color": "purple", "qty": 10, "vendor": ["C", "D"]}
+   {"_id": 6, "color": "pink", "qty": 5, "vendor": ["C"]}
+
+
+.. _bitwise:
+
+Bitwise
+-------
+
+The bitwise operator methods include:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Bitwise Method
+     - Matches
+      
+   * - ``bitsAllSet()``
+     - documents when the specified bits of a field are set (i.e. 1).
+
+   * - ``bitsAllClear()``
+     - documents when the specified bits of a field are clear (i.e. 0).
+      
+   * - ``bitsAnySet()``
+     - documents when at least one of the specified bits of a field are set (i.e. 1).
+     
+   * - ``bitsAnyClear()``
+     - documents when at least one of the specified bits of a field are clear (i.e. 0).
+
+The following example matches documents that have a bitField field
+with bits set at positions of the corresponding bitmask 50 (i.e.
+00110010) in the ``binary_numbers`` collection: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin bitsComparison
+   :end-before: end bitsComparison
+
+The BSON renders as:
+
+.. code-block:: json
+      :copyable: false
+   
+   { 
+      "a": 
+         { 
+            "$bitsAllSet": [ 1, 5 ] 
+         } 
+   }
+
+The output of the above code snippet should look something like this:
+   
+.. code-block:: json
+   :copyable: false
+      
+   { _id: 9, a: 54, binaryValue: "00110110" }
+   { _id: 12, a: 102, binaryValue: "01100110" }
+
+.. _geospatial:
+
+Geospatial
+----------
+
+The geospatial operator methods include:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Geospatial Method 
+     - Matches
+      
+   * - ``geoWithin()``
+     - documents containing a GeoJSON geometry field that falls within within a bounding GeoJSON geometry.
+
+   * - ``geoWithinBox()``
+     - documents containing a grid coordinates field that exist within the specified box.
```

Comment:
I'm not familiar with the "grid coordinates field" term -- can you point me to where they're referred to as such? Or did you mean just "coordinates", or a specific type like a Point?

Also, I wouldn't say the "field" exists within the specified box but rather the "value" exists. Comment applies to all instances.